utils/utils.go

```diff
@@ -88,3 +89,34 @@ func DirName() (string, error) {
 func IsOption(confirm, short, long string) bool {
 	return strings.EqualFold(confirm, short) || strings.EqualFold(confirm, long)
 }
+
+type Color struct {
+	Red   int64
+	Green int64
+	Blue  int64
+}
+
+func NewColor(hex string) (*Color, error) {
```

Comment:
The logic is not rocket science, but I guess it wouldn't hurt to add a test with some basic values and some basic errors.